### PR TITLE
(WIP) Update Travis to include gometalinter

### PR DESCRIPTION
**Pull request by seiflotfy**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,9 +4,17 @@ go:
...
 install:
  - go get github.com/constabulary/gb/...
+ - go get github.com/alecthomas/gometalinter
+ - gometalinter --install --update 
 
 # before_script:
  - make dist
 
 script:
+ - gometalinter -D gocyclo ./src/config/...
```
Comment (njpatel) [inline .travis.yml]: `gometalinter -D gocyclo ./src/...` ?

Comment (seiflotfy) [inline .travis.yml]: Can't because it wil lfail on datamodel due to datamodel/pb

**Comment by dgryski** [comment]
I run `go vet` as part of my CI for my projects; `go lint` has too many false positives, and I would assume many of the things flagged as part of gometalinter might also not all be *required* to be fixed.

**Comment by seiflotfy** [comment]
@dgryski its not what is flagged as much as it is an issue with the gopath due to the vendoring and stuff

**Pull request merged**